src/vs/workbench/contrib/search/browser/anythingQuickAccess.ts

```diff
@@ -87,8 +87,6 @@ export class AnythingQuickAccessProvider extends PickerQuickAccessProvider<IAnyt
 
 		picker: IQuickPick<IAnythingQuickPickItem> | undefined = undefined;
```

Comment:
It was moved out because the `EditorViewStateManager` has disposables and should be disposed of on dispose of its parent. We could also just dispose it from the parent's parent if it's still in pickstate. Would you rather have that?

Reply:
I do not see an issue with keeping it inside even if disposable, can you clarify? I can help move it back if there is trouble.

Reply:
Perhaps we can just make pickstate disposable also, then just call `dispose` on pickstate when it is time- that's a good point. I can move it back.

Reply:
👍